servicedef/service_params.go

```diff
@@ -11,9 +11,8 @@ const (
 	CapabilityAllFlagsClientSideOnly             = "all-flags-client-side-only"
 	CapabilityAllFlagsDetailsOnlyForTrackedFlags = "all-flags-details-only-for-tracked-flags"
 
-	CapabilityPersistentDataStore = "persistent-data-store"
-	CapabilityBigSegments         = "big-segments"
-	CapabilityTags                = "tags"
+	CapabilityBigSegments = "big-segments"
+	CapabilityTags        = "tags"
 )
```

Comment:
Q: why was persistent-data-store removed?

Reply:
The earlier addition of it was a mistake - I haven't implemented that feature yet.